src/services/communications/Ethernet_packets.py

```diff
@@ -0,0 +1,52 @@
+import struct
+
+
+class Packets:
+    def __init__(self,*packets): 
```

Comment:
pointers in python? I don't get the `*` before packets, is it some language feature I'm not aware of?

Reply:
I used it to receive a number not know of parameters, also used in serialize. if only receives one parameter with a list of more than one packet it can be called as packets = Packets(*list_packets). I think it has versality in case you use it like:
packet = Packets(packet1,packet2,packet3...)

Reply:
ah yes, I had an aneurism yesterday, didn't remember that was the syntax